frontend/class-opengraph.php

```diff
@@ -576,7 +576,10 @@ public function image( $image = false ) {
 		$opengraph_images = new WPSEO_OpenGraph_Image( $this->options, $image );
 
 		foreach ( $opengraph_images->get_images() as $img ) {
-			$this->og_tag( 'og:image', esc_url( $img ) );
+		    $tag_suffix = 0 === strpos($img, 'https://')
```

Comment:
Thank you for your pull request. 

I have one point of feedback.

Can you make this line more readable. For example:
```
$og_image_tag = 'og:image';
if ( 0 === strpos( $img, 'https://' ) {
    $og_image_tag .= 'secure_url';
} 

$this->og_tag( $og_image_tag, esc_url( $img ) );
```